data/heliarch intro.txt

```diff
@@ -0,0 +1,1570 @@
+			`	"You were seen landing on our worlds and rings, bringing about strange scanning mechanisms, and were also found to be scanning some of our ships. Why is that?"`
+			choice
+				`	"I was just curious about your species."`
+					goto lie
+				`	"What are you talking about? I never did any of that. Must have been my cousin."`
+					goto lie
+				`	"Scanning? I'm sorry, I think my ship just had some malfunctions, I didn't mean to offend."`
+					goto lie
+				`	"The Heliarchs asked me to do it."`
+					goto truth
+			label lie
+			`	Around ten seconds pass without it saying anything, simply staring at your eyes and blinking slowly.`
+			`	"I see."`
+			`	Without another word, it walks away, headed to the Wardragon.`
+				decline
+			label truth
+			`	"And why did you agree? Do you wish to join their ranks, perhaps?" It asks.`
+			choice
+				`	"I just wanted to help, it's not like I destroyed your ships or something."`
+					goto justhelp
+				`	"They told me that the Quarg keep others in 'separate boxes', and that you hinder their development."`
+					goto hinder
+			label justhelp
+			`	"Perhaps not, but you agreed to collect intel on us and our worlds, which hold countless civilians, and deliver the findings to a group that hates us."`
+				goto end
+			label hinder
+			`	"And you believe that? Have we restricted your movement? Chastised you for interacting with the Hai, or Korath? Even after you agreed to help those foolish, tyrannical Heliarchs, are you not here, in one of our worlds, peacefully conversing with a Quarg?"`
+			label end
+			`	It blinks a few more times, as a few small stripes on it's face rapidly alternate between shades of deep blue and bright red, then ends with, "It would be most wise to cut off all your ties with these 'Heliarchs,' human."`
+			`	It walks away, headed to the Wardragon it looked at before.`
+				decline
+
+
+
+mission "Heliarch Recon 3-A"
+	name "Preparations for Scanning"
+	description "Head to <destination>, where the Heliarchs will lend you one of their scanners."
+	deadline
+	landing
+	source
+		government "Heliarch"
+		not planet "Ring of Wisdom"
+	destination "Ring of Wisdom"
+	to offer
+		has "Quarg Interrogation: offered"
+		has "Rim Archaeology 5B: active"
+		has "event: rim archaeology results"
+		not "Resistance: Questions: offered"
+		"reputation: Quarg" >= 1
+		"coalition jobs" >= 65
+	on offer
+		conversation
+			`Upon landing, a trio of Heliarchs walk up to your ship to meet with you.`
+			`	They inform you that though they looked over the scan logs of the Quarg ships, they weren't capable of detecting anything different from back when they were actively at war against them.`
+			`	"Although tinkered with, the human scanners sadly could not give us the precise information we needed."`
+			`	"Diminute stashes of their outfits, our ancestors stole, but never to fully capture one of the Quarg ships, did they manage," the Kimek adds.`
```

Comment:
```suggestion
			`	"Diminutive stashes of their outfits, our ancestors stole, but never to fully capture one of the Quarg ships, did they manage," the Kimek adds.`
```
`Diminute` is so rarely used it shows as a spelling error for me. It might be better to use a word readers are more likely to be familiar with.